common/uint256.go

```diff
@@ -10,15 +10,21 @@ import (
 
 const UINT256SIZE = 32
 
-var EmptyUint256 Uint256
-
 type Uint256 [UINT256SIZE]uint8
 
+var EmptyUint256 Uint256
+var MaxUint256 = Uint256{
+	0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
+	0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
+	0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
+	0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
+}
+
 func (u *Uint256) CompareTo(o Uint256) int {
 	x := u.ToArray()
 	y := o.ToArray()
 
-	for i := len(x) - 1; i >= 0; i-- {
+	for i := 0; i < len(x); i++ {
```

Comment:
Uint256.CompareTo change from descending cmp to ascending cmp, means Uint256 store BigEndian changed to LittleEndian.
Uint160 should be changed also for consistency, and additional highlight srcCode's comments and API docs.

Reply:
Also changed Uint160